.github/workflows/generate-zip.yml

```diff
@@ -0,0 +1,49 @@
+name: Generate ZIP
```

Comment:
@peterwilsoncc These files were added because we intentionally include them: https://github.com/10up/distributor/blob/develop/package.json#L70

Can we remove them?